frontend/app_flowy/lib/workspace/presentation/widgets/float_bubble/question_bubble.dart

```diff
@@ -40,6 +46,69 @@ class QuestionBubble extends StatelessWidget {
                 case BubbleAction.help:
                   _launchURL("https://discord.gg/9Q2xaN37tV");
                   break;
+                case BubbleAction.debug:
+                  final deviceInfoPlugin = DeviceInfoPlugin();
+                  final deviceInfo = deviceInfoPlugin.deviceInfo;
+                  
+                  deviceInfo.then((info) {
+                    var debugText = "";
+                    info.toMap().forEach((key, value) {
+                      debugText = debugText + "$key: $value\n";
+                    });
+
+                    Clipboard.setData(ClipboardData( text: debugText ));
+
+                    Widget toast = Container(
+                      padding: const EdgeInsets.symmetric(horizontal: 24.0, vertical: 12.0),
+                      decoration: BoxDecoration(
+                        borderRadius: BorderRadius.circular(25.0),
+                        color: theme.main1,
+                      ),
+                      child: Row(
+                      mainAxisSize: MainAxisSize.min,
+                      children: const [
+                          Icon(Icons.check),
+                          SizedBox(
+                          width: 12.0,
+                          ),
+                          Text("Copied debug info to clipboard!"),
```

Comment:
Rats, I should have seen this the first time. My bad.  The text should use the translation mechanism. Note that it is not up to you to add translations (you can, nobody is stopping you...) As a developer you just have to make sure that the code uses translations.  The translators will do the translating in time...